packages/calypso-e2e/src/jest-playwright-config/global-setup.ts

```diff
@@ -5,7 +5,27 @@ import { TestAccount } from '../lib/test-account';
 import pwConfig from './playwright-config';
 
 export default async (): Promise< void > => {
-	const { AUTHENTICATE_ACCOUNTS } = envVariables;
+	const { AUTHENTICATE_ACCOUNTS, CALYPSO_BASE_URL } = envVariables;
+
+	try {
+		const response = await fetch( CALYPSO_BASE_URL, {
```

Comment:
`fetch` is available natively in Node 22 🎉 
https://nodejs.org/docs/latest-v22.x/api/globals.html#fetch